Integrations/integration-Office_EWS.yml

```diff
@@ -793,8 +870,14 @@ script:
         }
         fetchArgs.last_time = new Date(lastRun.lastTime).toISOString();
         fetchArgs['target-mailbox'] = getDefaultTargetMailbox();
+        if (params.folder) {
+            fetchArgs['folder'] = lastRun.folderId;
+            fetchArgs['target-folder'] = replaceInTemplates(targetFolderTmpl,fetchArgs);
+        } else {
+         fetchArgs['target-folder'] = replaceInTemplates(defaultTargetTargetFolderTmpl,fetchArgs);
+        }
         fetchArgs.version = params.defaultServerVersion
-        raw = sendRequest(fetchTmpl, fetchArgs);
+        raw = sendRequest(fetchTmpl, fetchArgs.'Envelope.Body.FindItemResponse.ResponseMessages.FindItemResponseMessage.-ResponseClass');
```

Comment:
I think you accidentally wrote `.` instead `,` (between `fetchArgs` & `Envelope..`)